_includes/patterns-filter.html

```diff
@@ -253,13 +253,19 @@
   // get all empty filter containers
   const container = document.getElementById("compatibility-div");
   const categoryContainer = document.getElementById("category-div");
-  
+
+  // globalTechnologyList stores all the technologies checked by user as strings
+  // example: ["gcp", "keda"]
+  let globalTechnologyList = [];
+
+  // globalCategorySelected stores the category selected by the user since only one can be selected at a time we don't need any array
+  // initially it is set to "All"
+  let globalCategorySelected = "All";
```

Comment:
In similar wane we can support multiple category selection, can you ensure that we do that too

Reply:
I can implement that feature, for that I will have to convert those radio buttons to checkboxes, shall I go ahead and do that?

Reply:
Hey I have updated the branch, we can now filter while selecting multiple categories! @Yashsharma1911

Reply:
Thankyou Kanishk